instrumentation/ktor/ktor-2-common/library/src/main/kotlin/io/opentelemetry/instrumentation/ktor/v2_0/common/client/AbstractKtorClientTracingBuilder.kt

```diff
@@ -159,13 +160,14 @@ abstract class AbstractKtorClientTracingBuilder(
    *
    * @param emitExperimentalHttpClientMetrics `true` if the experimental HTTP client metrics are to be emitted.
    */
-  @Deprecated("Please use method `emitExperimentalHttpClientMetrics`")
+  @Deprecated("Please use method `Experimental.emitExperimentalHttpClientMetrics`")
   fun setEmitExperimentalHttpClientMetrics(emitExperimentalHttpClientMetrics: Boolean) {
     if (emitExperimentalHttpClientMetrics) {
       emitExperimentalHttpClientMetrics()
     }
   }
 
+  @Deprecated("Please use method `Experimental.emitExperimentalHttpClientMetrics`")
```

Comment:
I'm not sure this is needed as the class itself is deprecated. Also doesn't `Experimental` only work with the new telemetry builder class?

Reply:
I was trying to provide a hint where this method went since it's not obvious otherwise